pallets/eth-bridge/src/lib.rs

```diff
@@ -868,3 +872,57 @@ pub mod pallet {
         }
     }
 }
+
+impl<T: Config> Pallet<T> {
+    pub fn has_collator_casted_event_vote(author_id: T::AccountId) -> bool{
+        has_author_casted_event_vote::<T>(&author_id)
```

Comment:
we can eliminate `has_collator_casted_event_vote` and use `has_author_casted_event_vote` directly in the runtime.